src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/GetPackageReferenceUtility.cs

```diff
@@ -60,38 +60,30 @@ internal static PackageIdentity UpdateResolvedVersion(LibraryDependency projectL
         }
 
         /// <summary>
-        /// Gets the dependencies of a top level package and caches these if the assets file has changed
+        /// Updates the transitive dependencies for the project.
```

Comment:
This comment looks less descriptive than what we had before. Consider describing the method in a way that is not already achieved by the name of the method.